### PR TITLE
fix word asynchronous

### PR DESCRIPTION
**Pull request by Aissaoui-Ahmed**
Refs: #87 
         [Omar alharbi](https://twitter.com/loboly_19/status/1486191149161267204)

### CODE REVIEW

**Comment by atefBB** [comment]
cc @Aissaoui-Ahmed what's up about this? See my question above!

**Comment by Aissaoui-Ahmed** [comment]
@atefBB where!

**Comment by atefBB** [comment]
> @atefBB where!

See this link https://github.com/softvenue/i18n/pull/88#discussion_r793966907

File: i18n.json [hunk 1]
```diff
@@ -208,9 +208,9 @@
     {
         "term": "ASYNCHRONOUS",
         "wAR": [
-            "تزامني",
-            "متزامن",
-            "متواقت"
+            "غير تزامني",
+            "غير متزامن",
+            "غير متواقت"
```
[the remark this answers is not in the document]
Comment (Aissaoui-Ahmed) [reply i18n.json:213]: <div dir=rtl>من ميقات على حسب علمي</div>

**Pull request merged**